### ISSUE TITLE
Click-focus for empty frames

### ISSUE THREAD

**Issue by The-Compiler** [herbstluftwm/herbstluftwm#20]
We should be able to focus empty frames with click, or with the mouse if focus_follows_mouse is on

**Comment by jfchevrette** [comment]
+1
I would also like to be able to click empty frames to select them even though focus_follows_mouse is off

**Comment by jmcblane** [comment]
Is there any hope for this?

**Comment by t-wissmann** [comment]
Yes, definitely. And if someone finds time to fix this I'm happy to merge it.
